Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ui/SmartLiveData.kt

```diff
@@ -38,7 +48,42 @@ class SmartLiveDataProperty<T : Any>(
     }
 }
 
-class SmartLiveData<T : Any>(
+class SmartSingleLiveEvent<T : Any>(
```

Comment:
Hint hint .. are we using LiveData for things it is not designed for ;)

Reply:
We kind of are, I removed this class, it was not used.